### PR TITLE
feat(plugin): common event format logs

### PR DESCRIPTION
**Pull request by algchoo**
### Proposed Change

Ported CEF plugin over from Stanza project.

example logs
```
Aug 02 15:25:11 localhost CEF:0|nxlog.org|nxlog|2.7.1243|Executable Code was Detected|Advanced exploit detected|9|src=192.168.255.110 spt=46117 dst=172.25.212.204 dpt=80
Aug 02 15:25:08 <local0.info> 10.217.31.247 CEF:0|Citrix|NetScaler|NS10.0|APPFW|APPFW_STARTURL|8|src=10.217.253.78 spt=53743 method=GET request=http://vpx247.example.net/FFC/login.html msg=Disallow Illegal URL. cn1=233 cn2=205 cs1=profile1 cs2=PPE0 cs3=AjSZM26h2M+xL809pON6C8joebUA000 cs4=ALERT cs5=2012 act=blocked
Aug 02 15:25:10 zurich CEF:0|security|threatmanager|1.0|100|worm successfully stopped|5|src=10.0.0.1 dst=2.1.2.2 spt=1232 
Aug 02 15:25:10 zurich CEF:0|security|threatmanager|1.0|100|worm successfully stopped|3|src=10.0.0.1 dst=2.1.2.2 spt=1232 
```

gcp logs screenshot
gcp-cef-logs

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: plugins/common_event_format_logs.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        - type: severity_parser
+          if: 'attributes.severity != nil'
+          parse_from: attributes.severity
+          preset: none
+          mapping:
+            info:
+              - min: 0
+                max: 3
+              - low
+            warning:
```
Comment (BinaryFissionGames) [inline plugins/common_event_format_logs.yaml:70]: List of otel aliases is here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/types/severity.md#how-severity-mapping-works
```suggestion
            warn:
```

File: plugins/common_event_format_logs.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              - min: 4
+                max: 6
+              - medium
+            error:
+              - min: 7
+                max: 8
+              - high
+            critical:
```
Comment (BinaryFissionGames) [inline plugins/common_event_format_logs.yaml:78]: Maybe this should be `fatal`? If not, then probably `error4`. (critical is not an otel mapping)
```suggestion
            fatal:
```

File: plugins/common_event_format_logs.yaml (hunk 1 above)
Comment (algchoo) [reply plugins/common_event_format_logs.yaml]: I'll have to keep that in mind, appreciate the link!

File: plugins/common_event_format_logs.yaml (hunk 2 above)
Comment (algchoo) [reply plugins/common_event_format_logs.yaml]: I believe fatal makes sense here.

**Review by BinaryFissionGames — APPROVED** [review]

**Pull request merged**